src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionTestResultViewModel.swift

```diff
@@ -16,7 +16,8 @@ class ExposureSubmissionTestResultViewModel {
 		onContinueWithSymptomsFlowButtonTap: @escaping () -> Void,
 		onContinueWarnOthersButtonTap: @escaping (@escaping (Bool) -> Void) -> Void,
 		onChangeToPositiveTestResult: @escaping () -> Void,
-		onTestDeleted: @escaping () -> Void
+		onTestDeleted: @escaping () -> Void,
+		onTestCertificateTapped: @escaping(HealthCertificate, HealthCertifiedPerson) -> Void
```

Comment:
Just realized, would be more in line with the others if named like this

```suggestion
		onTestCertificateCellTap: @escaping(HealthCertificate, HealthCertifiedPerson) -> Void
```